v1.1/sql-statements.md

```diff
@@ -95,6 +95,17 @@ Statement | Usage
 [`SHOW SESSIONS`](show-sessions.html) | List details about currently active sessions.
 [`SHOW QUERIES`](show-queries.html) | List details about current active SQL queries.
 
+## Jobs Management Statements
+
+Jobs in CockroachDB represent tasks that might not complete immediately, such as schema changes or enterprise license backups or restores.
```

Comment:
I think elsewhere you say "enterprise backups or restores" (i.e., without "license"), but totally up to you.